webcompat/webhooks/model.py

```diff
@@ -0,0 +1,178 @@
+        """
+        payload_request = self.prepare_accepted_issue()
+        public_number = WebHookIssue.get_public_issue_number(self.public_url)
+        # Preparing the proxy request
+        path = f'repos/{PUBLIC_REPO}/{public_number}'
+        proxy_response = make_request('patch', path, payload_request)
+        return proxy_response
+
+    def prepare_accepted_issue(self):
+        """Create the payload for the accepted moderated issue.
+
+        When the issue has been moderated as accepted,
+        we need to change a couple of things in the public space
+
+        - Title
+        - Body
+        - Any labels from the private issue
+        """
+        # Gets the labels from the body
+        labels = get_issue_labels(self.body)
+        labels.extend(self.original_labels)
+        # Let's remove action-needsmoderation in case it's here
+        if 'action-needsmoderation' in labels:
+            labels.remove('action-needsmoderation')
+        # Prepares the payload
+        payload_request = {
+            'labels': labels,
+            'title': self.title,
+            'body': self.body
+        }
+        return payload_request
+
+    def prepare_public_comment(self):
+        """Build the comment for the public repo."""
+        # issue number on private repo
+        number = self.number
+        # public issue data
+        public_url = self.public_url
+        public_number = WebHookIssue.get_public_issue_number(public_url)
+        # prepare the payload
+        return f'[Original issue {public_number}]({public_url})'
+
+    def reject_private_issue(self):
+        """Send a rejected moderation PATCH on the public issue."""
+        payload_request = prepare_rejected_issue()
+        public_number = WebHookIssue.get_public_issue_number(
+            issue_info['public_url'])
+        # Preparing the proxy request
+        path = f'repos/{PUBLIC_REPO}/{public_number}'
+        proxy_response = make_request('patch', path, payload_request)
+        return proxy_response
+
+    def tag_as_public(self):
+        """Set the core actions on new opened issues.
+
+        When a new issue is opened, we set a couple of things.
+
+        - Browser label
+        - Priority label
+        - Issue milestone
+        - Any "extra" labels, set from GET params
+
+        Then Send a GitHub PATCH to set labels and milestone for the issue.
+
+        PATCH /repos/:owner/:repo/issues/:number
+        {
+            "milestone": 2,
+            "labels": ['Label1', 'Label2']
+        }
+        """
+        # Grabs the labels already set so they will not be erased
+        # Gets the labels from the body
+        labels = get_issue_labels(self.body)
+        labels.extend(self.original_labels)
+        milestone = app.config['STATUSES']['needstriage']['id']
+        # Preparing the proxy request to the public repo
+        path = f'repos/{PUBLIC_REPO}/{self.number}'
+        payload_request = {'labels': labels, 'milestone': milestone}
```

Comment:
This should be `payload_request = {'labels': labels, 'milestone': self.milestone}`